routes/register.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express');
+
+const router = express.Router();
+const encrypt = require('./encrypt');
+
+const User = require('../db/models/User');
+
+router.get('/', (req, res) => {
+  // TODO: render a register form
+  res.send('it is a register block');
+});
+
+
+router.post('/send', (req, res) => {
```

Comment:
pls create controller forlder and move there this callback

Reply:
created controller